CHANGES.rst

```diff
@@ -10,6 +10,8 @@ New Features
 
 - ``astropy.convolution``
 
+  - Major change in convolution behavior and keyword arguments (#5782)
```

Comment:
Maybe just mention "more details in the API changes section" or similar?  Right now if I looked at this I'd be like "what changes? I can't trust this thing!"
(also, period at the end.)